logging-sample/README.adoc

```diff
@@ -3,59 +3,99 @@
 
-Next to storing logging a log management provides much more functionallity.
-One important aspect is the support of custom fields / tags that can be used to analyze logging.
-The Rico context API provides an easy way to add metadata to an application and by using the Rico logging-appenders such metadata will automatically be added to log informations.
-Next to this the GELF standard is used in this sample to send the logging over an UDP connection to the used log manager.
+Next to storing logging a log management provides much more functionality.
+One important aspect is the support of custom fields or tags that can be used to analyze logging.
+The Rico context API provides an easy way to add metadata to an application.
+By using the Rico logging-appender such metadata will automatically be added to log information.
+Next to this the GELF standard is used in this sample to send the logging over a UDP connection to the used log manager.
 
 image::.readme/logging-appender.svg[]
 
 === Using the Graylog stack
 
-The sample provides docker containers to easily start a preconfigured Graylog instance that can will be used to store the logging.
-The complete setup can be found in the `docker-graylog` folder of the logging sample.
+The sample provides docker containers to easily start a preconfigured Graylog instance that is used to store the log messages.
+The complete setup can be found in the `docker-graylog` folder.
 The `docker-compose` tool is needed to run the sample.
 If `docker-compose` is installed on your system you only need to execute the `start.sh` from `docker-graylog` folder to boot the graylog instance.
 Once this is done the web ui of Graylog can be accessed at http://localhost:9000[http://localhost:9000].
-When opening the ui for the first time you need to login to Graylog (user: admin - passwort: admin).
 
+==== Configuring Graylog
 
-TODO: Short Description of the Graylog ui
+When opening the ui for the first time you need to login to Graylog (user: admin - passwort: admin).
 
-TODO: Define GELF Input in Graylog
+Graylog supports different sources of log messages.
+Before it is able to receive any message you need to tell it where to listen.
 
+1. Visit to http://localhost:9000[http://localhost:9000]
+1. Login to Graylog (user: admin - passwort: admin)
+1. In the top menu select "System" > "Inputs"
+1. Select "GELF UDP" as the input type and click "Launch new input".
+1. In the popup select the "global" checkbox and give the input a title (i.e. GELF UDP)
+1. Click "save".
++
 image::.readme/graylog-gelf-input.png[]
-
-Once Graylog is started the 2 Spring Boot application that are part of the sample can be started.
-Go to the `server-spring-log4j` or `server-spring-logback` subfolder and use maven to start the server:
-`mvn spring-boot:run`
+1. Click "Show received messages" to the right of the newly added input.
+1. Click on the little play symbol in the top right corner to enable auto refresh of the page.
```

Comment:
I assume that should be:
```suggestion
2. Click on the little play symbol in the top right corner to enable auto refresh of the page.
```

Reply:
no, this automatically renders to a correctly numbered list, no need to count by hand ;-)